asset_tracker_app/lib/utils/constants.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:asset_tracker_app/view/home_view.dart';
+import 'package:asset_tracker_app/view/login_screen_view.dart';
+import 'package:asset_tracker_app/view/onboard_view.dart';
+import 'package:asset_tracker_app/widgets/onboard_view/onboard_page_widget.dart';
+import 'package:flutter/material.dart';
+
+class ToScreen {
+  static const String loginPage = '/login';
+  static const String homePage = '/home';
+  static const String onboardPage = '/onboarding';
+}
+
+class AppRoutes {
+  static Map<String, Widget Function(BuildContext)> routes = {
+    '/onboarding': (context) => const OnboardingScreenView(),
+    '/login': (context) => const LoginScreenView(),
+    '/home': (context) => HomePageView(),
+  };
+}
+
+class BackgroundColors {}
+
+//OnBoard Page
+class OnboardingPages {
```

Comment:
bu listeyi başka bir dosyada tanımlayabiliriz. hatta mixin bile kullanabilirsin. title description için lokalleşme veya ortak bir yerden bilgilerin alınması lazım. direkt string yazmak sonrası için işini zora sokar. Bir şirkete bu projeye yaptığını hayal edelim ve sana bir gün gelip dediler ki, uygulama dili portekizceyi desteklesin. tüm sayfaları gezip if elseler mi koyacaksın?